functions/src/alchemy_webhook.ts

```diff
@@ -105,45 +115,42 @@ export const setRealTimeCashAddress = functions.https.onCall(
       );
     }
 
-    // https://docs.alchemy.com/reference/create-webhook
-    // create webhook for user with userId
+    // TODO: get webhoookId and X-Alchemy-Token from dashboard
     const options = {
-      method: "POST",
+      method: "PATCH",
       headers: {
-        Accept: "application/json",
-        "X-Alchemy-Token": "this-is-a-token", // TODO: get token from alchemy
-        "Content-Type": "application/json",
+        accept: "application/json",
+        "X-Alchemy-Token": "qewrewqrewqr",
+        "content-type": "application/json",
       },
       body: JSON.stringify({
-        network: "ETH_MAINNET",
-        webhook_type: "ADDRESS_ACTIVITY",
-        webhook_url: "localhost/{userId}", // TODO: replace with real webhook url
+        addresses_to_add: ["0x9131F047B512fD9cc42afc7025e2e9Cb7E62eedF"],
+        webhook_id: "qwereqwrqewreqw",
```

Comment:
What's this webhook id?

Reply:
it's the webhook created from dashboard, i just have some random letters there.